### PR TITLE
added link for linkedin school of sre

### PR DESCRIPTION
**Pull request by kai3x5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Contributions are always welcome!
 * [Making a Lion Bulletproof: SRE in Banking](https://www.infoq.com/presentations/ing-sre-teams-practices/)
 * [Identifying and tracking toil using SRE principles](https://cloud.google.com/blog/products/management-tools/identifying-and-tracking-toil-using-sre-principles)
 
-## Education
+## Education
```
Comment (dastergon) [inline README.md:129]:
```suggestion
## Education
```

**Comment by dastergon** [comment]
Hi @kai3x5,

Thanks for the contribution! Would it be easy to rebase your PR? It looks like it has conflicts.

Thanks!

**Comment by kai3x5** [comment]
My bad sure thing!

**Comment by dastergon** [comment]
@kai3x5 can you rebase from the CLI? Auto-resolve on Github adds an extra merge commit which is not useful for the purposes of the repo.

**Comment by kai3x5** [comment]
I just rebased from CLI now.

**Comment by kai3x5** [comment]
Should I close this pull request and just do a new pull request?

**Comment by dastergon** [comment]
Looks like the previous merge commit is still there, it'd be much easier I guess to create a new PR @kai3x5

**Comment by kai3x5** [comment]
I made a new pr. My bad @dastergon

**Pull request closed without merging**